tools/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ filegroup(
     testonly = True,
     srcs = [
         "BUILD.bazel",
+        "//tools/override-generator:all_files",
```

Comment:
The user-facing tools are under `cmd` and maintainer-facing tools are under `tools`.

Reply:
I see. I am not sure how much we want to open this up to be a public CLI and maintain it indefinitely. What do you think?